### ISSUE TITLE
File evaluation_fid does not exist

### ISSUE THREAD

**Issue by IceIce1ce** [qsx830/MixBridge#1]

**Comment by IceIce1ce** [comment]
I also found that  loss += 1*F.mse_loss(i, torch.full_like(i, 1/weight.shape[0])) is wrong. It should change to  loss += 1*F.mse_loss(i, torch.full_like(i, 1/len(weight))). Moreover, the evaluation.py cannot work with line run.generation(opt, val_dataset, corrupt_method, True)

**Comment by qsx830** [comment]
Hi,

You are right about the issues you've identified. I am sorry for the oversights before I release the codes.

I will update the codes to address these issues.

Best regards,
Shixi

**Comment by IceIce1ce** [comment]
Hi author, 

Congratulations on your ICML paper and your great code. Thanks for your information.
